web_development_101/project_html_css.md

```diff
@@ -94,6 +94,7 @@ There are a couple of ways to go about doing this, but the simplest is this:
 Submit a pull request with a link to your solution in this [file](https://github.com/TheOdinProject/curriculum/edit/master/web_development_101/project_html_css.md) when your project is complete. See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
 * Add your solution below this list
+* [Cristina Popescu's Solution (Easy)](https://cristinaap.github.io/google-homepage) - [View in browser](https://cristinaap.github.io/google-homepage/)
```

Comment:
Your first link should be to the github repository your project exists in so others can read your code and learn from it. Can you fix this? Then I'll merge it :)

Reply:
Sure, i changed the link and made a new pull request. Sorry...first time doing this :)

Reply:
Ok...saw afterwards that I can make the change directly...